app/views/users/show.html.erb

```diff
@@ -89,17 +89,32 @@
 
         <md-input-container>
           <label>Full Name</label>
-          <input ng-model="user.name" ng-change="user.changed = true">
+          <input name="fullname" required ng-model="user.name" md-maxlength="15" ng-change="user.changed = true">
```

Comment:
15 is different than 35, which is what is stated in the message.

The database will actually take a string up to 255 characters here. How do we know what is long enough here?